### PR TITLE
Write landing page

### PR DESCRIPTION
**Pull request by danifosy**
Did I forget to write my landing page on Friday? Yes. Yes I did. 

Deployment: https://a-dogs-life-landingpage-wsrsxx.herokuapp.com/

### CODE REVIEW

**Review by marwinburesch — COMMENTED** [review]
More like "White landing page" 😂 
image

File: src/app/components/SubmitButton/SubmitButton.tsx [hunk 1]
```diff
@@ -20,9 +28,36 @@ export default function SubmitButton({
...
+    case 'send':
+      iconElement = <SendIcon />;
+      break;
+    case 'signup':
+      iconElement = <SignUpIcon />;
+      break;
+    case 'upload':
+      iconElement = <UploadIcon />;
+      break;
+  }
```
Comment (marwinburesch) [inline src/app/components/SubmitButton/SubmitButton.tsx:57]: nice, but a map would be a bit more beautiful

**Review by marwinburesch — COMMENTED** [review]
🎉 herokuparty! 💯

File: src/app/components/SubmitButton/SubmitButton.tsx (hunk 1 above)
Comment (danifosy) [reply src/app/components/SubmitButton/SubmitButton.tsx:57]: Konni said the same last night and then I hissed at him like a cat with rabies. I will consider rewriting it if I have time :D

**Pull request merged**